docs/admonitions.rst

```diff
@@ -57,13 +57,33 @@ Version Directives
 ******************
 
 The `versionadded`, `versionchanged`, and `deprecated` directives defined by Sphinx are also
-rendered by the sphinx-immaterial theme as admonitions.
+overridden by the sphinx-immaterial theme to render as :rst:dir:`admonition`\ s with added
+functionality (like optionally making them :rst:`:collapsible:`).
 
-These admonition styles can be customized using the `Custom Admonitions`_ feature provided the
+Beware, these directive overrides do not support the :rst:`:title:` and
```

Comment:
This could be in a `.. warning`

Reply:
I know, but I didn't want to overuse admonitions in a doc the demonstrates admonitions (seemed like admonition litter).